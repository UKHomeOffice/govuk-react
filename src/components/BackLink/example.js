import React from "react";
import BackLink from ".";

export default () => <BackLink>Test</BackLink>;
