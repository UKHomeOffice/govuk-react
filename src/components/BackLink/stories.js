import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import BackLink from '.';

storiesOf('BackLink', module).add('Component default', () => (
  <BackLink goBack={action('go-back')}>Back</BackLink>
));
