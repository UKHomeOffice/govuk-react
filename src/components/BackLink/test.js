import React from "react";
import ReactDOM from "react-dom";
// import { shallow } from "enzyme";

import BackLink from "./";

describe("Back Link", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<BackLink>Example</BackLink>, div);
  });
});
