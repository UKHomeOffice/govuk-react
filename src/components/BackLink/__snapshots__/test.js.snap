// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Back Link matches snapshot: enzyme.mount 1`] = `
<BackLink
  goBack={[Function]}
>
  <glamorous(button)
    onClick={[Function]}
  >
    <button
      className="css-16uyu72"
      onClick={[Function]}
    >
      example
    </button>
  </glamorous(button)>
</BackLink>
`;

exports[`Back Link matches snapshot: enzyme.render 1`] = `
<button
  class="css-16uyu72"
>
  example
</button>
`;

exports[`Back Link matches snapshot: enzyme.shallow 1`] = `
<glamorous(button)
  onClick={[Function]}
>
  example
</glamorous(button)>
`;
